haiku-apps/paladin/paladin-2.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="An open source integrated development environment (IDE)"
+DESCRIPTION="Paladin is an open source integrated development environment (IDE) \
+modeled after BeOS' BeIDE.
+
+Paladin's feature set includes:
+
+- Command-line build support
+- Multithreaded builds
+- Revision control-friendly project files
+- More run options for projects
+- Explicit support for debugging with gdb under Haiku
+- Bundled helper tools
+- Streamlined project settings
+- Out-of-the-box support for Lex and Yacc
+- Support for text and binary resource files
+- Projects can include notes and other files that aren't source code
+- Project templates
+- Out-of-the-box makefile generation
+- Integrated source code management
+- 1-click project backups"
+HOMEPAGE="https://github.com/adamfowleruk/Paladin"
+COPYRIGHT="2001-2010 DarkWyrm
+       2014 John Scipione
+       2018 Adam Fowler"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/adamfowleruk/Paladin/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="f5e35acb174bb1ec8b0c2ba6940ccd0614642602d1eef13daed8b67308c3e941"
+SOURCE_FILENAME="paladin-$portVersion.tar.gz"
+SOURCE_DIR="Paladin-$portVersion"
+
+ARCHITECTURES="x86 x86_64"
+
+USER_SETTINGS_FILES="settings/Paladin_settings"
+
+PROVIDES="
+	paladin$secondaryArchSuffix = $portVersion
+	app:Paladin
+	app:SymbolFinder
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	devel:libpcre$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
```

Comment:
This should be a part of `BUILD_REQUIRES` instead.

Reply:
even though there's already a BUILD_REQUIRES above it? Do they need merging?

Reply:
Just tested this change. It breaks the build, so I intend to leave it as is